documentation/Windows-Installation-Guide.md

```diff
@@ -1,6 +1,14 @@
 
 # Windows Installation Guide
 
+## Table of Contents
+
+- [Installing Java](#installing-java)
+- [Installing Gradle](#installing-gradle)
+- [Installing Postgresql and Postgis](#installing-postgresql-and-postgis)
```

Comment:
Here we might want to replace move  Postgresql -> PostgreSQL and Postgis -> PostGIS. Same for their occurrence in the section.